microovn/ovn/services.go

```diff
@@ -0,0 +1,69 @@
+package ovn
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+
+	"github.com/canonical/microcluster/state"
+	"github.com/canonical/microovn/microovn/database"
+)
+
+func DisableService(s *state.State, service string) error {
+	err := s.Database.Transaction(s.Context, func(ctx context.Context, tx *sql.Tx) error {
+		exists, err := database.ServiceExists(ctx, tx, s.Name(), service)
```

Comment:
The `service` string is supplied by the end user of the program, we need to do some sort of validation of it before passing it on into the innards of the program.

Who knows what the user could craft to have the database or `snapctl` do unexpected things with elevated privileges.